AutomatedTesting/Gem/PythonTests/Atom/tests/MaterialEditor_Atom_BasicTests.py

```diff
@@ -140,37 +145,32 @@ def MaterialEditor_BasicFunctionalityChecks_AllChecksPass():
-        test_material_3_document_id = atom_tools_utils.open_document(os.path.join(test_data_path, test_material_3))
-        test_material_4_document_id = atom_tools_utils.open_document(os.path.join(test_data_path, test_material_4))
-        test_material_5_document_id = atom_tools_utils.open_document(os.path.join(test_data_path, test_material_5))
-        test_material_6_document_id = atom_tools_utils.open_document(os.path.join(test_data_path, test_material_6))
-        Report.result(
-            Tests.verify_all_documents_are_opened,
-            atom_tools_utils.is_document_open(test_material_1_document_id) is True and
-            atom_tools_utils.is_document_open(test_material_2_document_id) is True and
-            atom_tools_utils.is_document_open(test_material_3_document_id) is True and
-            atom_tools_utils.is_document_open(test_material_4_document_id) is True and
-            atom_tools_utils.is_document_open(test_material_5_document_id) is True and
-            atom_tools_utils.is_document_open(test_material_6_document_id) is True)
+        material_document_ids = []
+        test_materials = [
+            test_material_1, test_material_2, test_material_3, test_material_4, test_material_5, test_material_6]
+        for material in test_materials:
+            material_document_id = atom_tools_utils.open_document(material)
+            Report.result(
+                Tests.verify_all_documents_are_opened,
+                atom_tools_utils.is_document_open(material_document_id) is True)
+            material_document_ids.append(material_document_id)
 
         # 5. Use the CloseAllDocumentsExcept bus call to close all but one.
-        atom_tools_utils.close_all_except_selected(test_material_1_document_id)
+        atom_tools_utils.close_all_except_selected(material_document_ids[0])
         Report.result(
             Tests.close_all_opened_assets_except_one,
-            atom_tools_utils.is_document_open(test_material_1_document_id) is True and
-            atom_tools_utils.is_document_open(test_material_2_document_id) is False and
-            atom_tools_utils.is_document_open(test_material_3_document_id) is False and
-            atom_tools_utils.is_document_open(test_material_4_document_id) is False and
-            atom_tools_utils.is_document_open(test_material_5_document_id) is False and
-            atom_tools_utils.is_document_open(test_material_6_document_id) is False)
+            atom_tools_utils.is_document_open(material_document_ids[0]) is True and
+            atom_tools_utils.is_document_open(material_document_ids[1]) is False and
+            atom_tools_utils.is_document_open(material_document_ids[2]) is False and
+            atom_tools_utils.is_document_open(material_document_ids[3]) is False and
+            atom_tools_utils.is_document_open(material_document_ids[4]) is False and
+            atom_tools_utils.is_document_open(material_document_ids[5]) is False)
```

Comment:
A validation helper can help make this result report easier to read and maintain.